templates/default/idea.desktop.erb

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=IntelliJ IDEA Community Edition
```

Comment:
Since the cookbook allows the installation of Ultimate Edition as well, can we either remove `Community Edition` from this line, or make this name dynamic?